src/test/kotlin/com/example/Employee/services/BankAccountServiceTest.kt

```diff
@@ -0,0 +1,144 @@
+package com.example.Employee.services
+
+import com.example.Employee.controllers.request.BankAccountRequest
+import com.example.Employee.exceptions.EntityNotFoundException
+import com.example.Employee.models.BankAccount
+import com.example.Employee.models.Employee
+import com.example.Employee.repositories.BankAccountRepository
+import com.example.Employee.repositories.EmployeeRepository
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.jupiter.api.Test
+import java.util.*
+import org.amshove.kluent.`should be equal to`
+import org.amshove.kluent.invoking
+import org.amshove.kluent.shouldThrow
+
+
+internal class BankAccountServiceTest {
+    private val employeeRepository= mockk<EmployeeRepository>()
+    private val bankAccountRepository=mockk<BankAccountRepository>()
+    private val bankAccountService=BankAccountService(employeeRepository,bankAccountRepository)
+
+    @Test
```

Comment:
There are lot of tests in the service mentioning what the response status code should be. For example, tests like `should return 404`. It is not the responsibility of the service to decide the status code of the response, or any other characteristics of the http response. That is up to the controller.

Reply:
But while writing the code, we are throwing these from the service right?

Reply:
Exception != status code.

Reply:
Renamed those tests